docs/estela-cli/quickstart.md

```diff
@@ -38,7 +38,20 @@ Successful login. API Token stored in ~/.estela.yaml.
 ```
 
 This will save your estela API key to the file `~/.estela.yaml`, and it is
-needed to access projects associated with your account.
+needed to access projects associated with your account.  
+
+**_Note:_** If you have installed Estela locally run the following command to get the _EXTERNAL-IP_ for the django-api:
```

Comment:
Please change this line to: _If you have installed estela locally, run the following command to get the host of the estela api, do not forget to add `http://` at the beginning_.